src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -45,22 +45,23 @@ public IEnumerable<PackageVerifierIssue> Validate(PackageAnalysisContext context
                             if (AssemblyHelpers.IsAssemblyManaged(assemblyPath))
```

Comment:
I tried doing this but `AssemblyDefinition.ReadAssembly(Stream)` does not seem to work. It throws a `System.ArgumentException: Value does not fall within the expected range.` But works when an actual file path is specified. I googled it and it seems like a bug in cecil but not very sure.